.gitlab-ci-ias.yml

```diff
@@ -39,7 +39,7 @@ prep-cuda:
     - gpu
     - springdale7
   script:
-    - module load cudatoolkit/10.2 gcc/7.3.0
+    - module load gcc/7.3.0 cudatoolkit/11.1
```

Comment:
GCC 7.3.0 is very old... How hard would it be to update this?

Reply:
This machine is beyond my control. Nevertheless, this is actually less of a problem as nvcc itself doesn't support the most recent GCC versions anyway.